source/index.html.md

```diff
@@ -540,26 +527,103 @@ Exactly one payment method must be used.
   <dd>
     Amount in minor units of given currency (e.g. cents if in Euro).
   </dd>
+
   <dt>currency
     <span class="type">[A-Z]{3}</span>
   </dt>
   <dd>
     <a target="_blank" href="currencies.txt">3-letter currency code</a>. (Some exponents differ from ISO 4217.)
   </dd>
+
+  <dt>credential
+    <span class="type">(<code>storing</code>|<code>using</code>)</span>
+  </dt>
+  <dd>
+    Indicate if the payment credential (e.g. PAN and expiry) will be stored for
+    a later transaction where the payment credential is not provided by the
+    cardholder but collected from (encrypted) storage.
+    <br />
+    <code>storing</code>: The payment credential will be stored; may only be
+    stored if the authorization is approved.
+    <br />
+    <code>using</code>: The payment credential was already stored and are now
+    being used.
+    <br />
+    If <code>initiator</code> is <code>merchant</code> then
+    <code>storing</code> is invalid and <code>using</code> is assumed.
```

Comment:
Suggestion to be further polished and writting using bullets: "If `initiator` is `merchant` then: a) `storing` is invalid and will cause an input error, and b) `using` is assumed if `credential` is left out."

Also, add "if `credential` is left out" to the next line (554). Of course that's also just a suggestion.

Reply:
Fixed by e4a6aebaf5e53b32346e1cb0199460b395865f85